MapboxMobileEvents.xcodeproj/project.pbxproj

```diff
@@ -1171,6 +1171,7 @@
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				LLVM_LTO = NO;
```

Comment:
Were these changes in LTO intentional? Not sure if they're actually needed...

Reply:
Yeah, I needed to turn off LTO on debug builds -- they were preventing breakpoints from operating for some reason.